ivy_tests/test_ivy/test_frontends/test_numpy/test_creation_routines/test_from_existing_data.py

```diff
@@ -18,7 +18,7 @@
         max_dim_size=5,
     ),
     num_positional_args=helpers.num_positional_args(
-        fn_name="ivy.functional.frontends.numpy.asarray"
+        fn_name="ivy.functional.frontends.numpy.creation_routines.asarray"
```

Comment:
you should vary the `fn_tree` argument in `test_frontend_function`, which should be sampled from the combination of permitted namespaces. We should only have one unit test for this function, which should simply be the earliest of the two categories in the API. Currently we have two identical tests which is not what should happen.